### DISCUSSION TITLE
Hello there, I wanna rejoin please?

### DISCUSSION THREAD

**Discussion by ghost** [discordjs/discord.js#10406] [Discord Server Ban]
1202432891100069898 here my id and please? May I ask what I did to gets banned that worse?!

**Comment by almostSouji** [comment]
You keep switching between accounts, some of which seems to be motivated by loosing access to compromised accounts.

While we generally allow appeals for compromised accounts _(should they be restored through contacting Discord - not just creating alts when it happens)_, we see this as a security risk to our members if it happens at this volume.
As a result we have removed you from our Discord Community and are not granting appeals for this action.

Please note that re-joining Discord servers on alt accounts to bypass bans is against Discord ToS and can result in further action taken as per
- <https://discord.com/terms>
- <https://discord.com/guidelines>
